File: my-folder/1482-how-many-numbers-are-smaller-than-the-current-number/solution.cpp
class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) {
       
       std::vector<int> response(nums.size(), 0);

       for (int i = 0; i < nums.size(); ++i) {
            for (int j = 0; j < nums.size(); ++j) {
                if (nums[i] > nums[j]) {
                    ++response[i];
                }
            }

       }
       return response;

    }
};

File: my-folder/1482-how-many-numbers-are-smaller-than-the-current-number/solution.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
    
        sorted_nums = sorted(nums)

        count = {}

        for i, n in enumerate(sorted_nums):
            if n not in count:
                count[n] = i
        return [count[num] for num in nums]
            
